traitcuration/traits/datasources/zooma.py

```diff
@@ -0,0 +1,64 @@
+import requests
+
+from ..models import Trait, MappingSuggestion, OntologyTerm, User
+from .ols import make_ols_query, get_ontology_id
+
+
+BASE_URL = "http://www.ebi.ac.uk/spot/zooma/v2/api"
```

Comment:
I'm wondering if we should start collecting all base URLs and configuration settings in a separate module (`config`?) to make it easier in the future to make all of them customizable via a configuration page.

This is not necessary to do as part of this PR, just something to consider

Reply:
Hmm yes that seems like a good option, I will create an issue for that.